core/src/main/resources/spline.default.properties

```diff
@@ -36,6 +36,10 @@ spline.lineageDispatcher.http.className=za.co.absa.spline.harvester.dispatcher.H
 spline.lineageDispatcher.http.producer.url=
 spline.lineageDispatcher.http.timeout.connection=2000
 spline.lineageDispatcher.http.timeout.read=120000
+# disables protocol version negotiation and sets the version to 1.2 ( LATEST could be used as well)
+# spline.lineageDispatcher.http.apiVersion=1.2
```

Comment:
```suggestion
# Spline Producer API version to use.
# If unspecified, the protocol version is determined from the endpoint handshake response.
# If unspecified and a custom (non-Spline) endpoint is used, the Producer API version is automatically set to 1 (the oldest one supported).
# spline.lineageDispatcher.http.apiVersion=1 | 1.1 | 1.2 | LATEST
```

Reply:
> If unspecified and a custom (non-Spline) endpoint is used, the Producer API version is automatically set to 1 (the oldest one supported).

they still need the status endpoint to respond `200 OK` as it was before, but when implementing endpoints for plan and events this should be easy to add as well.

Reply:
By that I meant that custom endpoints would hardly send any Spline custom headers, and if one isn't sent, the version defaults to 1